src/helpers/sync_helpers.sh

```diff
@@ -18,11 +18,11 @@ function sync_branch_steps {
       echo "merge_tracking_branch"
       echo "merge $(parent_branch "$branch")"
     else
-      echo "rebase_tracking_branch"
+      echo "$PULL_BRANCH_STRATEGY""_tracking_branch"
```

Comment:
I believe they are
in my terminal if I do:
export FOO="meow"
echo "$FOO_foo"
it prints nothing because FOO_foo is not a variable
however
echo "$FOO""_foo" 
gives the desired result of meow_foo